core/region.go

```diff
@@ -246,14 +249,13 @@ func (r *region) scanForEnabledAutoScalingGroupsByTag(asgs *[]*string) {
 		logger.Println("Failed to describe AutoScaling tags in",
 			r.name,
 			err.Error())
-		return
 	}
+	return asgs
 }
 
 func (r *region) scanForEnabledAutoScalingGroups() {
-	asgNames := []*string{}
 
-	r.scanForEnabledAutoScalingGroupsByTag(&asgNames)
+	asgNames := r.scanForEnabledAutoScalingGroupsByTag()
```

Comment:
I would have renamed this method to something a bit different, than what this method is doing/named. Something like: `getEnabledAutoScalingGroups` for example, as the names of those methods is very similar but don't fulfil the same purpose, it's a bit confusing, and adding `ByTag` is a bit pointless, as it's currently the only way of enabling ASGs.